### ISSUE TITLE
intl-messageformat throws error when HTML included in message

### ISSUE THREAD

**Issue by superafroman** [kaisermann/svelte-i18n#120]
With the latest release and move to the latest intl-messageformat messages that include variables and HTML tags have started to throw errors.

This has been caused by a new option and functionality in intl-messageformat, `ignoreTag` that is defaulted to false and not exposed or set by svelte-i18n.

**Logs**
SyntaxError: Expected "#", "'", "\n", "{", argumentElement, double apostrophes, end of input, or tagElement but "<" found.

**To Reproduce**
I will open a PR and reference in comments.

**Expected behavior**
Messages with variables and HTML continue to behave as they have done and b rendered correctly.

**Stacktraces**
If you have a stack trace to include, we recommend putting inside a `<details>` block for the sake of the thread's readability:

<details>
  <summary>Stack trace</summary>

SyntaxError: Expected "#", "'", "\n", "{", argumentElement, double apostrophes, end of input, or tagElement but "<" found.
    at peg$buildStructuredError (svelte-i18n/node_modules/intl-messageformat-parser/src/parser.js:401:16)
    at Object.peg$parse [as pegParse] (svelte-i18n/node_modules/intl-messageformat-parser/src/parser.js:3109:15)
    at Function.parse [as __parse] (svelte-i18n/node_modules/intl-messageformat-parser/index.js:11:24)
    at new IntlMessageFormat (svelte-i18n/node_modules/intl-messageformat/src/core.js:138:42)
    at svelte-i18n/src/runtime/includes/formatters.ts:94:5
    at Object.memoizedFn (svelte-i18n/src/runtime/includes/memoize.ts:13:31)
    at Object.<anonymous> (svelte-i18n/test/runtime/includes/formatters.test.ts:201:7)
    at Object.asyncJestTest (svelte-i18n/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:106:37)
    at svelte-i18n/node_modules/jest-jasmine2/build/queueRunner.js:45:12
    at new Promise (<anonymous>)
    at mapper (svelte-i18n/node_modules/jest-jasmine2/build/queueRunner.js:28:19)
    at svelte-i18n/node_modules/jest-jasmine2/build/queueRunner.js:75:41
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

</details>

**Comment by cupcakearmy** [comment]
Same issue here, would like to do some text bold but it fails

**Comment by superafroman** [comment]
Hey @cupcakearmy, you can work around not having this option by escaping your HTML tags, e.g.

```
{
  someGreatLookupKey: `'<p>'Your account will be connected with '<strong>'{name}'</strong></p>'`
}
```

**Comment by SuPuL** [comment]
@superafroman Thx. Tried lots of things but missed this one.

**Comment by cupcakearmy** [comment]
@superafroman Thanks a lot! Will try that 😃

**Comment by kaisermann** [comment]
Fixed in 3.3.2 🎉
